lib/Constants.js

```diff
@@ -201,3 +201,21 @@ module.exports.ChannelTypes = {
     GUILD_NEWS: 5,
     GUILD_STORE: 6
 };
+
+module.exports.Intents = {
+    guilds:                 1 << 0,
+    guildMembers:           1 << 1,
+    guildBans:              1 << 2,
+    guildEmojis:            1 << 3,
+    guildIntegrations:      1 << 4,
+    guildWebhooks:          1 << 5,
+    guildInvites:           1 << 6,
+    guildVoiceStates:       1 << 7,
+    guildPresences:         1 << 8,
+    guildMessages:          1 << 9,
+    guildMessageReactions:  1 << 10,
+    guildMessageTyping:     1 << 11,
+    directMessages:         1 << 12,
+    directMessageReactions: 1 << 13,
+    directMessageTyping:    1 << 14
+};
```

Comment:
Would prefer to keep it as is, since it's the same format as we use for event names. It's formatted like that in Constants because intent integer calculating checks directly against Constants.

Reply:
See also: Permissions, which use camelCase. There's nowhere in the lib where a dev using the lib has to use Discord case, would not be the greatest idea to change that now.